src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint react/no-array-index-key: 0 */
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Accordion } from 'react-bootstrap'
+import './collapsible-section.scss'
+
+const CollapsibleSection = ({ header, description }) => (
+  <Accordion defaultActiveKey='0' className='simple-collapsible-section'>
+    <Accordion.Item eventKey='0' className='border-0 px-0'>
+      <Accordion.Header><h3>{header}</h3></Accordion.Header>
```

Comment:
what's a header we would be using in the webstore? can I see a demo of this using mocked (but real) data from the api so I see how this looks?

Reply:
the header will just be passed in from the webstore - the [webstore PR for this page](https://github.com/scientist-softserv/webstore/pull/88) shows how it would be used

Reply:
i just added another story using data from the webstore as well, here's a vid of it:
https://share.getcloudapp.com/4gu4OAmv

Reply:
also, is there a way to make this header a bit smaller? maybe an h5 or h6? it's not meant to be a real prominent part of the page.